app/forms/mappings/helpers/FormatAnswers.scala

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package forms.mappings.helpers
+
+object FormatAnswers {
+
+  def trimAndCompressSpaces(original: String): String                = original.trim.replaceAll("\\s+", " ")
+  def removeWhitespace(original: String): String                     = original.filterNot(_.isWhitespace)
+  def toUppercaseAndRemoveSpacesAndHyphens(original: String): String =
+    original.toUpperCase.replaceAll(" ", "").replaceAll("-", "")
+  def addHyphens(original: String): String = {
```

Comment:
pedantic - this formats a nirms number specifically, so name should probably reflect that?